ultra/ultra/baselines/common/replay_buffer.py

```diff
@@ -188,6 +189,13 @@ def add(
 
         self._memory.append(new_experience)
 
+    def __getitem__(self, index):
+        if torch.is_tensor(index):
+            index = index.tolist()
+        state, action, reward, next_state, done, others = tuple(self._memory[index])
+        next_state = torch.cat((state[1:], next_state))  # Reattach the previous frames.
+        return state, action, reward, next_state, done, others
```

Comment:
Hey @christianjans I was just wondering where do we use this method? I am trying to spot an value access from the replay buffer objects which will call the __getitem__()?